### ISSUE TITLE
Upgrade to GlusterFS 3.12

### ISSUE THREAD

**Issue by swade1987** [apprenda/kismatic#1080]
We should probably upgrade to the latest version of GlusterFS which seems to be 3.12

See [here](https://www.gluster.org/install/)

**Comment by fredzqm** [comment]
We have an issue with glusterfs https://bugzilla.redhat.com/show_bug.cgi?id=1433906.

This is fixed in 3.12.5, but right kismatic is still using 3.8, which is causing an issue.

**Comment by swade1987** [comment]
I have a PR for this https://github.com/apprenda/kismatic/pull/1081 but for some reason, it's failing the build, is the RPM URL we are using for the PR the same as the one that works for you?

**Comment by fredzqm** [comment]
Yeah, this is exactly the same issue I had. It would be great if this could be resolved.

**Comment by e100** [comment]
First time kistmatic user

The Ubuntu ppa for 3.8 no longer exists resulting in this error:

Configure Package Repositories                                                  [ERROR]
- Task: add Gluster deb repository
  axxxxxx: failed to fetch PPA information, error was: HTTP Error 404: Not Found[ERROR]
  bxxxxxx: failed to fetch PPA information, error was: HTTP Error 404: Not Found[ERROR]


I edited the ansible configs to use 3.10 repo and 3.10.12-ubuntu1~xenial1 package and was able to get the cluster installed
